.github/workflows/test.yml

```diff
@@ -82,7 +84,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          # check out full history
```

Comment:
this comment is for fetch-depth: 0

Reply:
oh woops, went trigger happy with a find/replace